lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -30,6 +30,7 @@ export default function AssignmentActivity() {
   );
   const students = useAPIFetch<StudentsResponse>(routes.students_metrics, {
     assignment_id: assignmentId,
+    public_id: dashboard.organization_public_id || [],
```

Comment:
If you rebase on `main`, you should now be able to drop `|| []`.

See https://github.com/hypothesis/lms/pull/6488